packages/devtools_app/lib/src/debugger/program_explorer_controller.dart

```diff
@@ -49,6 +52,12 @@ class ProgramExplorerController extends DisposableController
   final _initialized = ValueNotifier<bool>(false);
   bool _initializing = false;
 
+  /// The line corresponding with the currently selected outline node. Results
+  /// in the declaration of the selected outline node being highlighted in the
+  /// [CodeView].
+  ValueListenable<int> get focusedLine => _focusedLine;
+  final _focusedLine = ValueNotifier<int>(null);
```

Comment:
instead of a separate ValueNotifier for the line number, can we just have a ValueNotifier for the _outlineSelection and pull the line number from that value?